### PR TITLE
Move manifest constants to shared file

### PR DESCRIPTION
**Pull request by andreasgerstmayr**
Resolves: #139
Signed-off-by: Andreas Gerstmayr <agerstmayr@redhat.com>

### CODE REVIEW

**Review by rubenvp8510 — APPROVED** [review]

File: internal/manifests/manifestutils/constants.go [hunk 1]
```diff
@@ -6,4 +6,18 @@ const (
 	// nolint #nosec
 	// BearerTokenFile declares the path for bearer token file for service monitors.
 	BearerTokenFile string = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+
+	ConfigVolumeName = "tempo-conf"
```
Comment (iblancasa) [inline internal/manifests/manifestutils/constants.go:10]: Would it make sense to add doc about each variable?

**Review by iblancasa — APPROVED** [review]

File: internal/manifests/manifestutils/constants.go (hunk 1 above)
Comment (pavolloffay) [inline internal/manifests/manifestutils/constants.go:10]: Please add comments newly added public constants in this file

File: Makefile [hunk 2]
```diff
@@ -303,7 +303,7 @@ e2e:
 	$(KUTTL) test
 
 .PHONY: prepare-e2e
-prepare-e2e: kuttl start-kind cert-manager set-test-image-vars docker-build load-image-operator deploy
+prepare-e2e: kuttl start-kind cert-manager deploy-minio set-test-image-vars docker-build load-image-operator deploy
```
Comment (pavolloffay) [inline Makefile:306]: ` deploy-minio`

Does not seem to be related to this PR. Consider moving it to a separate PR.

Comment (iblancasa) [reply Makefile:306]: I think he merged #151 here. So, these changes will not be shown in this PR when #151 is merged.

Comment (andreasgerstmayr) [reply Makefile:306]: Yep, I had to merge the e2e-fixes from #151 into this branch to make the tests pass. It's been merged but somehow these changes are showing up here (maybe because the squash created a new git commit?), anyway, I'll rebase it.

File: internal/manifests/manifestutils/constants.go (hunk 1 above)
Comment (andreasgerstmayr) [reply internal/manifests/manifestutils/constants.go:11]: done

**Review by pavolloffay — APPROVED** [review]
